Library/Formula/srmio.rb

```diff
@@ -15,7 +15,7 @@ class Srmio < Formula
 
   def install
     if build.head?
-      system "chmod u+x genautomake.sh"
+      chmod 1, "genautomake.sh"
```

Comment:
I'm afraid `chmod 0001, "genautomake.sh"` won't give the desired effect. How did you come up with the `0001` value (that is identical to just `1`)? That's only giving executable permissions to “other” (but not to the owner or group of the file) and neither read nor write permissions to anyone. Or to express it symbolically, that would be equivalent to `chmod ug=,o=x genautomake.sh`.

Long story short, I think the best Ruby code we can come up with is giving this file the canonical permissions an executable file would get (read and execute for everyone, write only for the owner):

```ruby
chmod 0755, "genautomake.sh"
```

(The effect is not identical to the original code, but quite likely what was originally intended.)